### ISSUE TITLE
Confusing Error: chose string type for null column Detail

### ISSUE THREAD

**Issue by arky** [CJWorkbench/loadurl#3]
Trying to load API "https://covid19.th-stat.com/api/open/cases" endpoint results in following "chose string type for null column Detail". The 'Detail' column is empty and is correctly selected as text. 

![image](https://user-images.githubusercontent.com/808895/88757322-ecb47780-d18f-11ea-85e2-2dbaf239ba97.png)

**Comment by adamhooper** [comment]
Yes, this technical message should not make it to the user.. We should translate it into English (and then other languages).

The missing code should replace https://github.com/CJWorkbench/cjwparse/blob/6f44beecc0c6da78661cdac02a5114e3e3de06ca/cjwparse/json.py#L80-L85. We should model it after https://github.com/CJWorkbench/cjwparse/blob/6f44beecc0c6da78661cdac02a5114e3e3de06ca/cjwparse/csv.py#L35. The list of messages to parse is at https://github.com/CJWorkbench/arrow-tools#json-to-arrow.
